GETTING_STARTED.md

```diff
@@ -0,0 +1,30 @@
+The simplest way to start to use 'AppKid' is an example app like:
+
+https://github.com/smumriak/AppKidDemo
+
+...which you can [after installing dependances] simply git-clone to your local machine, and 'swift run' it.  
+
+git clone https://github.com/smumriak/AppKidDemo 
+cd AppKidDemo
+swift build 
+swift run
+
+This application has AppKit defined as a dependancy in its 'Package.swift', and will auto-download and auto-compile AppKid for you when it is 'swift build' or 'swift run' the first time.
+
+see here:
+https://github.com/smumriak/AppKidDemo/blob/3f617eaa9a69ddf2f0333552fd9ded9f1921aa7b/Package.swift#L10
+https://github.com/smumriak/AppKidDemo/blob/3f617eaa9a69ddf2f0333552fd9ded9f1921aa7b/Package.swift#L20
+
+
+This example app does *not* have a main.swift , and instead uses the @main attribute in Sources/AppDelegate.swift to achieve the same result.  In swift if your project provides main.swift it is assumed as "the main function", but its common for swift apps to be built without one [ using @main ], and the code that *would* have been in main.swift is autogenerated.
+
+for more details, please refer to the swift documentation.
+
+refs: 
```

Comment:
Since this is not a handwriting and something remotely resembling documentation you probably want to keep it on some level of readability and convenience, i.e. not using this kind of shortening.

Reply:
done